### ISSUE TITLE
pentile penalty

### ISSUE THREAD

**Issue by FossPrime** [LeaVerou/dpi#31]
The GS4 numbers are very misleading and against the purpose of the site. Pentile displays of the same dpi are inherently less sharp, by up to 19%. I would adjust the dpi to reflect that and put an asterisk on it.

http://www.bystander.info/2012/05/pentile-math.html

**Comment by LeaVerou** [comment]
Good point. I think it’s best to have both values, with the adjusted one in parentheses and a footnote to explain. A FAQ item would be a good idea too, providing a simple short explanation and linking back to that article.
